src/classes/sequelize-service.ts

```diff
@@ -40,47 +40,47 @@ import { ICreateSession } from '../interfaces/create-session';
 import { IDeleteSession } from '../interfaces/delete-session';
 
 @injectable()
-export class SequelizeService<A, CP extends {} = {}> extends Service implements ISequelizeService<A, CP> {
-  private primaryKeyField: keyof A = 'id' as keyof A;
-  protected computedPropertiesManager: IComputedPropertiesManager<A, CP>;
+export class SequelizeService<W, R extends W, C extends {} = {}> extends Service implements ISequelizeService<W, R, C> {
```

Comment:
Should `W` extend `{}` ?

Reply:
Same for `C`?

Reply:
See: https://github.com/bluebirds-blue-jay/sequelize-service/commit/76123161a6908b18f7644ae6dfd1dccedc8943a8